Ginger/GingerCore/ALM/Qtest/ExportToQtest.cs

```diff
@@ -0,0 +1,594 @@
+                                    if (publishToALMConfig.ToAttachActivitiesGroupReport)
+                                    {
+                                        if ((activGroup.TempReportFolder != null) && (activGroup.TempReportFolder != string.Empty) &&
+                                            (System.IO.Directory.Exists(activGroup.TempReportFolder)))
+                                        {
+                                            //Creating the Zip file - start
+                                            string targetZipPath = System.IO.Directory.GetParent(activGroup.TempReportFolder).ToString();
+                                            string zipFileName = targetZipPath + "\\" + TestCaseName.ToString() + "_GingerHTMLReport.zip";
+                                            
+                                            if (!System.IO.File.Exists(zipFileName))
+                                            {
+                                                ZipFile.CreateFromDirectory(activGroup.TempReportFolder, zipFileName);
+                                            }
+                                            else
+                                            {
+                                                System.IO.File.Delete(zipFileName);
+                                                ZipFile.CreateFromDirectory(activGroup.TempReportFolder, zipFileName);
+                                            }
+                                            System.IO.Directory.Delete(activGroup.TempReportFolder, true);
+                                            //Creating the Zip file - finish
+                                            //Attaching Zip file - start
+                                            AttachmentFactory attachmentFactory = (AttachmentFactory)run.Attachments;
+                                            TDAPIOLELib.Attachment attachment = (TDAPIOLELib.Attachment)attachmentFactory.AddItem(System.DBNull.Value);
+                                            attachment.Description = "TC Ginger Execution HTML Report";
+                                            attachment.Type = 1;
+                                            attachment.FileName = zipFileName;
+                                            attachment.Post();
+
+                                            //Attaching Zip file - finish
+                                            System.IO.File.Delete(zipFileName);
+                                        }
+                                    }
+
+
+                                    //create run with activities as steps
+                                    run.CopyDesignSteps();
+                                    run.Post();
+                                    StepFactory stepF = run.StepFactory;
+                                    List stepsList = stepF.NewList("");
+
+                                    //int i = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)